modules/eloqua_api_auth_fallback/src/Commands/EloquaAuthTokensGenerate.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Drupal\eloqua_api_auth_fallback\Commands;
+
+use Drupal\Core\Config\ConfigFactory;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\eloqua_api_redux\Service\EloquaApiClient;
+use Drush\Commands\DrushCommands;
+
+/**
+ * A drush command for eloqua api authentication module.
```

Comment:
See above comment for writing better description!

Reply:
Updated